lib/client/client_store.go

```diff
@@ -249,3 +252,11 @@ func LoadKeysToKubeFromStore(profile *profile.Profile, dirPath, teleportCluster,
 	}
 	return kubeCert, privKey, nil
 }
+
+func noCredentialsError(err error) error {
+	return trace.Wrap(err, "no credentials")
+}
+
+func isNoCredentialsError(err error) bool {
+	return strings.Contains(err.Error(), "no credentials")
```

Comment:
nit (as it's private): What if `err` is `nil`? 🙈

Reply:
This should be ok now that it is using `errors.Is` to perform the check.